components/button/LinkButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from './Button';
+import { getClasses } from '../../helpers/component';
+
+const LinkButton = ({ children, className, ...rest }) => {
+    return (
+        <Button className={getClasses('pm-button-link', className)} {...rest}>
```

Comment:
When we started to write component, we started to use `getClasses` but finally it's not a good idea because the code is super simple as you can see:

```suggestion
        <Button className={`pm-button-link ${className}`} {...rest}>
```